wp-content/themes/core/components/header/subheader/css/subheader.pcss

```diff
@@ -25,13 +27,16 @@
 /* CASE: Page with featured image or a background color */
 .c-subheader--has-background {
 	background: var(--color-grey-light);
-	color: var(--color-white);
 
 	@media (--viewport-medium) {
 		align-items: flex-end;
 	}
 }
 
+.c-subheader--has-image {
+	color: var(--color-white);
+}
+
```

Comment:
Instead of setting text color in this case, please add the css class `t-theme--light` to main subheader classes, or even just the content wrapper's classes.

Reply:
A neat. TIL.